Utilities/WebAppCleanUp/WebAppCleanUp.groovy

```diff
@@ -66,8 +68,9 @@ def setup(String[] args) {
 		System.exit(0)
 	}
 
-    // load user properties file
-   	properties.load(new File("${getScriptDir()}/user.properties"))
+	// if specified, load the specified properties file
+	if (opts.prop)
+   		properties.load(new File("${opts.prop}"))
```

Comment:
This only allows to pass a property file with an absolute path.

The above description mentions also to provide a relative path. 

There is an sample a bit further below with the pwFile.

Reply:
The current code support both absolute paths and paths relative to the current working dir (which is different than the path relative to the dir where the script is located, as coded for the pwFile). What do you prefer?

Reply:
I've changed the process to use path relative to the location of the script. It is now consistent with the pwFile and other scripts that can be used in the pipeline.